imports/mail/MailgunMail.js

```diff
@@ -34,7 +34,17 @@ export class MailgunMail extends Mail {
         }
 
         // Send the request
-        Meteor.http.post(postURL, options); // do not pass callback so the post request will run synchronously
+        const result = HTTP.post(postURL, options); // do not pass callback so the post request will run synchronously
+        this._verifyStatus(result);
+    }
+
+    _verifyStatus(result) {
+        if (result.data && result.data.message === 'Queued. Thank you.') {
```

Comment:
Is this returned by the mailgun service? If so we should not test that as it may change at any time.

Test HTTP status codes and the existence of data, but not specific strings unless it is guaranteed that they don't change.

Reply:
I totally see your point. Unfortunately, we cannot check any status code since we get an http 200 in either case: success and error 🤷‍♂️. We could check if the attribute `id` is given.

Since we sent the emails asynchronously and currently do not notify the user if the mail could not be sent I want to notify the sysadmin in any case something is different than expected.

Reply:
I see. Just checked the Mailgun API and there really seems to be no reliable way to check for success.